app/models/user.rb

```diff
@@ -3,6 +3,9 @@ class User < ActiveRecord::Base
   has_many :likes
   has_many :comments
 
+  validates :name, presence: true
+  validates :posts_count, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
   def recent_posts
```

Comment:
- [ ] @binyamolango Great work adding all the validation for post and user models, however, you forgot to add tests for all of your models' methods and validations. Kindly consider adding these tests and making sure they are all passing.